packages/web/src/components/search/addons/Mic.js

```diff
@@ -74,7 +74,7 @@ class Mic extends React.Component {
 			}
 			this.instance = new SpeechRecognition();
 			this.instance.continuous = true;
-			this.instance.interimResults = true;
+			this.instance.interimResults = false;
```

Comment:
Cool, this looks like the issue, were we stopping the mic as soon as we got the result? I think we forgot to add the check for whether the response received is final or not.

Reply:
basically `onresult` gets called on each word results in a sentence. And we were setting values in the state only for first word and stopping the mic. But now `onresult` won't get called unless and until we stop to speak.

Reply:
Cool makes sense 👍